projects/forge-lib/src/lib/shared/form-editor-components/concrete/currency-editor/display/currency-editor-display.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, OnInit } from '@angular/core';
+import { CurrencyComponent } from '../../../../form-components/concrete/currency/currency.component';
+import { FormEditorDisplayComponent } from '../../../abstract/form-editor/display/form-editor-display-component';
+
+@Component({
+  selector: 'forge-renderer-currency-editor-display',
```

Comment:
no comments

Reply:
doesn't look like the file got updated with comments here